### ISSUE TITLE
sentinel-2 L2A vs L1C

### ISSUE THREAD

**Issue by vishalned** [ServiceNow/geo-bench#9]
Hey,

Just wanted to say, thanks for the great paper and repository. I am working on something in this same area and really appreciate the common utilities for benchmarking. My question is regarding the source of the individual datasets. 

If we consider only the Sentinel-2 benchmarking among the whole list, many of the datasets comprise of either L2A or L1C. Is it not a common remote sensing understanding that you perform well on L1C downstream tasks while considering L1C pre-training data, and the same for L2A as well. 

Just wondering if you put any thoughts into this, since you do not speak about this in the paper, and i wonder if you assume they are all similar (when normalised).

Would be great to get your thoughts on this .

**Comment by recursix** [comment]
Hi Vishal,

Thanks for reaching out, and I appreciate your thoughtful insights on our paper and repository. 

It's a fair point for L2A vs L1C. At the minimum, I should have put the information in BandInfo so that a model could condition on this information to be able to work well in zero-shot or few-shot. 

That said, let's speculate on the behavior of a large model in a fine-tuning regime. First, I believe that if we would make a classifier of L2A vs L1C, we could get really high accuracy (e.g. >99). If this is true, this also means that a model pre-trained on a mixture of L2A vs L1C would learn to infer this variable and adjust its behavior if it is useful for the predictions on the pre-training dataset. Finally, the normalization and fine-tuning happening on the downstream task would compensate and adapt for the lack of information. So I believe that a model pre-trained with only L2A or L1C could be slightly penalized, but a model trained on both L2A and L1C should work great.

But hey, that's just speculations.

**Comment by vishalned** [comment]
Well thats something I am working on.. so, lets see. 

Thanks for the response anyways.
